src/james.crypto.md5.clss.pas

```diff
@@ -52,13 +57,37 @@ implementation
 { TMD5Stream }
 
 function TMD5Stream.GetStream: IDataStream;
+{$IFNDEF FPC}
+  function getMd5HashString(Value: string): string;
```

Comment:
@nunopicado let's create a new class that will work with MD5 hash to refactor this code. Thus, we can will use this new class all over the code that might need MD5 hash. 
What do you think?